atomic

```diff
@@ -213,6 +213,37 @@ if __name__ == '__main__':
     imagesp.add_argument("--prune", action="store_true",
                          help=_("prune dangling images"))
 
+    # atomic migrate
+    migratep = subparser.add_parser("migrate", help=_("migrate docker from one backend storage "
+                                                    "to another"))
+    migrate_subparser = migratep.add_subparsers(help=_("migrate commands"))
+
+   # atomic export
+    exportp = migrate_subparser.add_parser(
+        "export", help=_("export a docker instance into a filesystem directory"),
+        epilog="export a docker instance. "
```

Comment:
export all containers/images/volumes.

Lets try to eliminate the word docker wherever possible.  There is potential this tool could be used in the future for non docker container tools like rocket or systemd-nspawn.